doctrine2/BaseMigration.php

```diff
@@ -0,0 +1,45 @@
+<?php
+declare(strict_types=1);
+
+namespace hiqdev\reodb\doctrine2;
+
+use PDO;
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+abstract class BaseMigration extends AbstractMigration
+{
+    /**
+     * @var bool
+     */
+    protected bool $attrEmulatePrepares;
+
+    /**
+     * @inheritDoc
+     */
+    abstract public function up(Schema $schema): void;
+
+    public function enableBatchSql():void
```

Comment:
```suggestion
    public function enableBatchSql(): void
```

Похоже, у тебя в штормне на настроен автоформаттер